standard/clause_0_front_material.adoc

```diff
@@ -27,6 +32,34 @@ The next-simplest use case is to put data values for each variable (parameter) i
 
 A sophisticated use case is to use tiling objects, where the data values are partitioned spatially and temporally, so that a single variable's data values would be split among several documents. A simple example of this would be to encode each timestep of a dataset in a separate file, but the tiles could also be divided spatially, like a tiled map server.
 
+[keywords]
+== Keywords
+
+The following are keywords to be used by search engines and document catalogues:
+
+ogcdoc, OGC Document, JSON, JSON-LD, CoverageJSON, CovJSON, Coverage, spatiotemproal, linked-data.
+
+[source]
+== Source of this document
+
+The majority of the content in this OGC document is a direct copy of the content originally contained at https://github.com/covjson/specification . No normative changes have been made to the content. This OGC document does contain content not in the original source CovJSON GitHub repository. Specifically, while derived from content of the original CovJSON repository, the Abstract, Keywords, Preface, Security Considerations, Submitting Organizations, Endorsers, Terms and Definitions, and References sections and Annex B (Bibliography) in this document are not found on the original CovJSON.
```

Comment:
Should there be a space after the GitHub URL?

Reply:
Also, should we be using the full name "CoverageJSON". I do use "CovJSON" a lot but mainly informally.